File: Module 5/KilometerConverter.cpp
/*****
 * Author: Glowstudent
 * Program: Kilometer Converter
 *****/

#include <iostream>
#include <string>

using namespace std;

void kilometerToMiles(float kilometers, float &miles);

int main()
{
    float kilometers, miles;

    cout << "Enter the number of kilometers to convert to miles: ";
    cin >> kilometers;

    kilometerToMiles(kilometers, miles);

    cout << kilometers << " kilometers is approximately " << miles << " miles." << endl;

    return 0;
}

void kilometerToMiles(float kilometers, float &miles)
{
    miles = kilometers * 0.6214;
}

File: Module 5/KilometerConverter.java
/*****
* Author: Glowstudent
* Program: Kilometer Converter
*****/

import java.util.Scanner;

public class KilometerConverter {

    static float convertToMiles(float km) {
        return km * 0.6214f;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        float km;
        float miles;

        System.out.print("Enter the number of kilometers to convert to miles: ");
        km = sc.nextFloat();

        miles = convertToMiles(km);

        System.out.printf("%s kilometers is approximately %.4g miles.", km, miles);
    }
}